packages/domain/generators.json

```diff
@@ -70,17 +70,15 @@
     }
   },
   "generators": {
-    "init": {
-      "factory": "./src/generators/init/init.compat#initSchematic",
-      "schema": "./src/generators/init/schema.json",
-      "description": "Initializes the @nrwl/angular plugin.",
-      "aliases": ["ng-add"],
-      "hidden": true
+    "ngProject": {
+      "factory": "./src/generators/create/angular/generator",
+      "schema": "./src/generators/create/angular/schema.json",
+      "description": "create angular project generator"
     },
-    "create": {
-      "factory": "./src/generators/create/generator",
-      "schema": "./src/generators/create/schema.json",
-      "description": "create generator"
+    "reactProject": {
+      "factory": "./src/generators/create/react/generator",
```

Comment:
The generators inside src are currently the exposed ones, but the plan is to replace them once the new generators are ready.